test/common_quantization.py

```diff
@@ -51,7 +52,6 @@ def test_only_train_fn(model, train_data, loss_fn=_default_loss_fn):
             optimizer.zero_grad()
             output = model(data)
             loss = loss_fn(output, target)
-            loss.backward()
```

Comment:
Why is this line removed? Without it backprop will not occur.

Reply:
oh, removed when debugging, will add back